modin/core/dataframe/pandas/partitioning/partition_manager.py

```diff
@@ -1227,6 +1227,7 @@ def compute_part_size(indexer, remote_part, part_idx, axis):
         for row_idx, row_values in enumerate(row_partitions_list):
             row_blk_idx, row_internal_idx = row_values
             col_position_counter = 0
+            row_offset = 0
```

Comment:
uh? is this really needed?

Reply:
This is necessary when trying to index with slice(None). `row_offset` is usually set in a for loop below this line, and then used outside the for loop. This causes `row_offset` to be referenced before assignment when the for loop has 0 iterations.